lizard_auth_server/tests/test_views.py

```diff
@@ -40,6 +43,32 @@ def test_smoke_as_admin(self):
         self.assertEquals(result.status_code, 200)
 
 
+class ConfirmDeletionUserconsentViewTestCase(TestCase):
+
+    def setUp(self):
+        self.factory = RequestFactory()
+        self.user = factories.UserF()
+        self.client = OIDC_Client.objects.create(client_id='id')
+        expires_at = datetime(3022, 2, 17)
+        date_given = datetime(3018, 4, 16)
+        self.user_consent = UserConsent.objects.create(
+            expires_at=expires_at, client=self.client, user=self.user, date_given=date_given)
+
+        self.userconsents_before_deletion = UserConsent.objects.count()
+
+    def test_remove_one_and_correct_user_from_UserConsent(self):
+        pk = self.user_consent.pk
+        request = self.factory.post('/confirm_deletion_userconsent/')
+        # Login
+        request.user = self.user
+        # Call the delete method
+        response = ConfirmDeletionUserconsentView.as_view()(request, pk=pk)
+        self.assertEqual(response.status_code, 302)
+        userconsents_after_deletion = UserConsent.objects.count()
+        self.assertEqual(1, self.userconsents_before_deletion - userconsents_after_deletion)
+        self.assertFalse(UserConsent.objects.filter(pk=pk).exists())
```

Comment:
Hier lijkt het me wel OK om meerdere asserts in één test te doen.

In het *algemeen* wil je dit in principe niet doen. Bij de eerste error faalt hij namelijk gelijk. En dat terwijl het eigenlijke probleem misschien duidelijker zou zijn bij een van de asserts die na die eerste komen. Dat zie je *wel* als het losse tests zijn.